cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionUtils.java

```diff
@@ -235,6 +248,12 @@ static ConcurrencyAdjustingRecommendation recommendedConcurrency(Map<BrokerEntit
       }
     }
 
+    if (allBrokersWithinAdjusterLimit) {
```

Comment:
Would you explain why do we want to increase cluster concurrency instead of broker concurrency when all brokers within adjuster limit?

Is that because concurrency adjuster tries its best to raise the broker concurrency, but still not able to hit the adjuster limit, meaning the cluster concurrency is too low currently?